docs/tutorial/first_dataset.markdown

```diff
@@ -0,0 +1,107 @@
+# Building Your First Dataset
+
+With a fully-built `QuantGov` corpus and an estimator, it's now possible
+to build a full set of analyses into a dataset. Generally that will mean
+running a number of natural language analyses as well as a number of
+
+## Running NLP Analyses
+
+As discussed [earlier](./natural_language_analysis.markdown),
+
+Let's start by running two built-in QuantGov analyses: the standard word
+count, and the conditional count, which gives one measure of the
+complexity of the document. Run these two commands:
+
+``` {.bash}
```

Comment:
I see {.bash} again, this must be something I am not picking up on.

Reply:
See above.